src/main/java/ua/com/conductor/config/SecurityConfig.java

```diff
@@ -0,0 +1,30 @@
+package ua.com.conductor.config;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+@EnableWebSecurity
+public class SecurityConfig extends WebSecurityConfigurerAdapter {
+
+    @Autowired
+    public void configureGlobal(AuthenticationManagerBuilder auth) throws Exception {
+        auth.inMemoryAuthentication().passwordEncoder(getEncoder())
+                .withUser("bobbob@bob.bo").password(getEncoder().encode("pass1word")).roles("USER");
+    }
+
+    protected void configure(HttpSecurity http) throws Exception {
+        http.authorizeRequests().anyRequest().authenticated().and().formLogin().permitAll()
```

Comment:
better to write each method on the new line:
```suggestion
        http.authorizeRequests()
        .anyRequest().authenticated()
        .and()
        .formLogin().permitAll()
```